src/universal/containers/app/MyAccountPage/PersonalInformationForm.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component, PropTypes } from 'react';
+import {reduxForm} from 'redux-form';
+import { FormTextField, RaisedButton } from './../../../components';
+
+@reduxForm({
+  form: 'PersonalInformationForm',
+  fields: [
+    'firstName',
+    'lastName',
+    'companyName',
+    'location'],
+})
+export default class PersonalInformationForm extends Component {
+  static propTypes = {
+    fields: PropTypes.object.isRequired,
+    handleSubmit: PropTypes.func.isRequired
+  };
+
+  render() {
+    const {
+      fields: {
+        firstName,
+        lastName,
+        companyName
+      },
+      handleSubmit
+    } = this.props;
+
+    const isSaving = false;
+
+    return (
+      <div>
+        <form onSubmit={handleSubmit}>
+          <div>
+            <FormTextField field={firstName}
+              type="text"
+              labelText="First name"
+              fullWidth
+              disabled={isSaving}/>
```

Comment:
`isSaving` is set to false? `isSaving` should be set from the reducer and handled here